modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs

```diff
@@ -110,14 +117,16 @@ public static void OpenFile(string slnPath, string scriptPath, int line)
                 args.Add("--line");
                 args.Add((line + 1).ToString()); // https://github.com/JetBrains/godot-support/issues/61
             }
+
             args.Add(scriptPath);
             try
             {
                 Utils.OS.RunProcess(path, args);
             }
             catch (Exception e)
             {
-                GD.PushError($"Error when trying to run code editor: JetBrains Rider. Exception message: '{e.Message}'");
+                GD.PushError(
+                    $"Error when trying to run code editor: JetBrains Rider. Exception message: '{e.Message}'");
```

Comment:
I would prefer to avoid making these code style changes, I assume these were made automatically by your IDE when saving the file.